src/calendar-list/item.js

```diff
@@ -47,8 +47,16 @@ class CalendarListItem extends Component {
           disabledByDefault={this.props.disabledByDefault}
           showWeekNumbers={this.props.showWeekNumbers}
           renderArrow={this.props.renderArrow}
-          onPressArrowLeft={this.props.onPressArrowLeft}
-          onPressArrowRight={this.props.onPressArrowRight}
+          onPressArrowLeft={this.props.onPressArrowLeft ? this.props.onPressArrowLeft : (_,month)=>{
```

Comment:
Using arrow functions in props is not advisable, you should move the functionality to separate methods. 
Also, you should check if 'this.props.calendarListContainer' is not undefined before invoking its method (if you use a callback, as I suggested in the first comment, you can instead call it safely using Lodash's 'invoke'). 
Third, you shouldn't assign to an argument, declare a new const for 'month' clone.